src/components/Cta/CtaLink.tsx

```diff
@@ -79,7 +83,7 @@ export const CtaLink = React.forwardRef<HTMLAnchorElement, CtaLinkProps>(
         {...rest}
         {...sbLinkProps}
         ref={ref}
-        href={myLinkWithUtm}
+        href={myLink}
```

Comment:
I originally wanted to pass the utm stuff to external link in case they link out to Giving or Make a Gift, but on second thought, I'll hold off

Reply:
Yeah, I don't think we want to do that. Usually the codes would be for our sites and specific campaigns but it probably doesn't hurt to add them.

Reply:
Sounds good to me 😄